simplecpp.cpp

```diff
@@ -2426,8 +2426,16 @@ std::map<std::string, simplecpp::TokenList*> simplecpp::load(const simplecpp::To
             continue;
 
         std::ifstream fin(filename.c_str());
-        if (!fin.is_open())
+        if (!fin.is_open()) {
+            if (outputList) {
+                simplecpp::Output err(fileNumbers);
+                err.type = simplecpp::Output::MISSING_HEADER;
```

Comment:
Should I better use `simplecpp::Output::ERROR` here?

Reply:
After looking at how Cppcheck handles the `outputList` I think it does make more sense to use `ERROR` here. I have changed it.